gcn/notices/chime/frb.schema.json

```
@@ -0,0 +1,88 @@
+{
+  "$id": "https://gcn.nasa.gov/schema/main/gcn/notices/chime/frb.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "CHIME/FRB Alert",
+  "description": "Fast radio burst alert from the Canadian Hydrogen Intensity Mapping Experiment.",
+  "type": "object",
+  "allOf": [
+    { "$ref": "../core/Alert.schema.json" },
+    { "$ref": "../core/Reporter.schema.json" },
+    { "$ref": "../core/Event.schema.json" },
+    { "$ref": "../core/DateTime.schema.json" },
+    { "$ref": "../core/Localization.schema.json" }
+  ],
+  "properties": {
+    "alert_kind": {
+      "type": "string",
+      "description": "The kind of alert (detection or subsequent). Similar meaning to alert_type"
+    },
+    "known_source": {
+      "type": "string",
+      "description": "Transient Name Server name of the known source (if a repeating FRB source)"
+    },
+    "snr": {
+      "type": "number",
+      "description": "Signal-to-noise ratio of the burst"
```

Is this different from `rate_snr` in core/Statistics.schema.json? 
```suggestion
      "description": "Signal-to-noise ratio of the burst [dimensionless]"
```

---

Though they are very similar, I think it would be misleading to call it "rate_snr" in the context of fast radio bursts. In the high energy regime, rates is usually a measure of photon counts / time, but for radio transients, rates are usually a measure of bursts / time. In other words, a "rate_snr" for a fast radio burst might mislead people into thinking there were multiple bursts detected.

---

We appreciate that rate_snr as used in high-energy is different than what you have here, but could you make it `[something]_snr`, `[something]_duration` so that it has a parallel structure to `rate_snr` and `image_snr`?